### PR TITLE
Fixes df-1461/ Event List

### PR DESCRIPTION
**Pull request by sebworks**
Added the Event list pattern 

## Additions

- Added events-docs.less
- Added event-post-macros.html
- Added events-posts.html

## Removals

- None

## Changes

- Updated the object passed to `post_related.render()`
- Updated the list of related links

## Testing

Navigate to
- events/spring-2014-rulemaking-agenda/

## Review

- @jimmynotjim  
- @sebworks 

## Preview
event-list

## Notes

- Wasn't Able to convert UTC to local for time element
- Calendar Icon/Title needs some cleanup

### CODE REVIEW

File: _includes/event-post-macros.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+   date:           A valid date.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro calendar_icon(calendar_date, additional_classes) %}
+    <time class="calendar-icon float-media" data-month="{{ calendar_date | date('%b') }}" data-day="{{ calendar_date | date('%d') }}" datetime="{{ calendar_date }}">
```
Comment (anselmbradford) [inline _includes/event-post-macros.html]: The float should probably be added outside of the macro, or added conditionally through a parameter, so the macro implementation can decide whether to float left or right or not at all (this is without looking into `float-media`—but I assume this adding a left float).

Comment (sebworks) [inline _includes/event-post-macros.html]: @anselmbradford, definitely shouldn't be inside the macro.

File: _includes/events-posts.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+{% macro render(posts) %}
+
+{% import "event-post-macros.html" as event_post_macros with context %}
+
+{% for post in posts%}
+
+    <article class="post-preview event-post-preview">
```
Comment (jimmynotjim) [inline _includes/events-posts.html]: `event-post-prevew` should be `post-preview__event` to follow the [CFPB flavor of BEM](http://cfpb.github.io/capital-framework/contributing/#follow-our-css-naming-conventions).

File: static/css/event-docs.less [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    .event-meta_venue:after {
+        content: "\2014";
+        margin-left: .5em;
+
+        .respond-to-max(@mobile-max {
+            content: none;
+        });
+    }
+
+    .summary_header {
```
Comment (jimmynotjim) [inline static/css/event-docs.less]: Do this and the `summary` prefixed items below need to be nested under `.summary`? They already include that in the block.

Comment (sebworks) [inline static/css/event-docs.less]: I think the 'event-meta' prefixed classes do but I'll move the 'summary' prefixed classes.

**Pull request merged**